hikari/events/reaction_events.py

```diff
@@ -258,6 +292,23 @@ def emoji_id(self) -> typing.Optional[snowflakes.Snowflake]:
             `builtins.None`.
         """
 
+    def is_for_emoji(self, emoji: typing.Union[emojis.Emoji, str], /) -> bool:
+        """Get whether the reaction event is for a specific emoji.
+
+        Parameters
+        ----------
+        emoji : typing.Union[hikari.emojis.Emoji, builtins.str]
+            The emoji to check.
+
+            Passing `builtins.str` here indicates a unicode emoji.
+
+        Returns
+        -------
+        builtins.bool
+            Whether the emoji is the one which was removed.
+        """
+        return emoji.id == self.emoji_id if isinstance(emoji, emojis.CustomEmoji) else emoji == self.emoji_name
```

Comment:
This could be moved to `ReactionEvent` to avoid code duplication

Reply:
reaction event doesn't have any emoji properties on it

Reply:
Should it 🤔

Reply:
It can't because ReactionDeleteAllEvent event doesn't have a relevant emoji